content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -306,10 +306,13 @@ The `init` command does not interpret any existing files. Instead, it creates a
 
 #### Arguments
 
-| Argument                | Required | Description                                                                                                                                                            | Default      |
-| ----------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
-| `--datasource-provider` | No       | Specifies the default value for the `provider` field in the `datasource` block. Options are `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`. | `postgresql` |
-| `--url`                 | No       | Define a custom datasource url.                                                                                                                                        |              |
+| Argument                | Required | Description                                                                                                                                                                                    | Default                       |
+|-------------------------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
+| `--datasource-provider` | No       | Specifies the default value for the `provider` field in the `datasource` block. Options are `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`.                         | `postgresql`                  |
+| `--url`                 | No       | Define a custom datasource url.                                                                                                                                                                |                               |
+| `--generator-provider`  | No       | Define the default generator provider to use.                                                                                                                                                  | `prisma-client-js`            |
+| `--preview-features`    | No       | Define the default Preview features to use. [Learn more](/concepts/components/preview-features)                                                                                                |                               |
```

Comment:
Is this the only one that can be used multiple times to define the array? If so, we should mention that somehow as it is different from the other arguments that are singular.

Reply:
Yes. I've created a small PR to take care of that with an example: https://github.com/prisma/docs/pull/5403